Sources/Realtime/RealtimeChannelV2.swift

```diff
@@ -81,6 +83,7 @@ public final class RealtimeChannelV2: Sendable {
   }
 
   /// Subscribes to the channel
+  @MainActor
```

Comment:
This isn't a breaking change since the method is `async`, the call site already have to call this method with an `await` independent of the running actor.